porchlightapi/sources/github.py

```diff
@@ -85,16 +135,47 @@ def github_data(project_url, branch='master', commit=''):
             'commit_changes': commit_changes,}
 
 
-def github_source(repository, branch='master'):
+def github_commit_source(repository, branch='master'):
     """
-    Github value source for Porchlight.
+    Github value source for the latest commit on a branch.
```

Comment:
This is such a small change, but it's this kind of attention to detail -- even in comments -- that is great to see.